### PR TITLE
[2.0.1] Update version.props and add list of dependendencies that are also being updated

### PR DESCRIPTION
**Pull request by natemcmaster**
CI will fail until we actually have a candidate build of the EF dependencies.

:watch: will wait until those are available to merge

### CODE REVIEW

File: src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>A compatibility layer for sharing identity databases between Microsoft.AspNet.Identity.EntityFramework and Microsoft.AspNetCore.Identity.EntityFrameworkCore.</Description>
-    <VersionPrefix>0.3.0</VersionPrefix>
+    <VersionPrefix Condition="'$(ExperimentalProjectVersionPrefix)' != ''">$(ExperimentalProjectVersionPrefix)</VersionPrefix>
```
Comment (JunTaoLuo) [inline src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj]: Why the experimental project version prefix? I'd assume that we would never decide during a patch update to designate a pre-release package as rtm. In fact, in the past, we do not ship patch updates for pre-release packages at all.

Comment (natemcmaster) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj]: Yeah, idk. @Tratcher ? We shipped 0.3.0, to nuget.org so I assume we should also ship 0.3.1.

File: version.props [hunk 2]
```diff
@@ -1,6 +1,10 @@
-<!-- This file may be overwritten by automation. Only values allowed here are VersionPrefix and VersionSuffix.  -->
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>2.0.1</VersionPrefix>
+    <ExperimentalProjectVersionPrefix>0.3.1</ExperimentalProjectVersionPrefix>
```
Comment (natemcmaster) [inline version.props]: just FYI - I've found it's too easy to forget to update a VersionPrefix set in csproj directly. I'm hoping putting all versions in version.props will help us when we have multiple version prefixes coming from one repo.

File: src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj (hunk 1 above)
Comment (JunTaoLuo) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj]: cc @Eilon would we want to ship updates for pre-release packages in 2.0.x patches? We decided not to do this in 1.x patches.

File: version.props (hunk 2 above)
Comment (JunTaoLuo) [reply version.props]: Ah that's why you moved it here. 👍

File: src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj (hunk 1 above)
Comment (Tratcher) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj]: @haok

Comment (HaoK) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj]: I don't think we need to update this package, it never was officially released and is more of a prototype/sample anyways.

Comment (Eilon) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj]: Yeah in the past we didn't ship patches of these, so no point starting now. Less stuff to deal with.

Comment (natemcmaster) [reply src/Microsoft.AspNet.Identity.AspNetCoreCompat/Microsoft.AspNet.Identity.AspNetCoreCompat.csproj]: Ok. I'll mark this package as "noship" in the packages.csv list. That is what ultimately determines what will ship.

**Pull request merged**